<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StationSelectForm</class>
 <widget class="QWidget" name="StationSelectForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1018</width>
    <height>563</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="7">
    <widget class="QPushButton" name="btnAdd">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>90</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>Добавить</string>
     </property>
    </widget>
   </item>
   <item row="2" column="9">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="3" colspan="7">
    <widget class="StationWidget" name="selectStationWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="isMapButtonVisible" stdset="0">
      <bool>false</bool>
     </property>
     <property name="isSearchVisible" stdset="0">
      <bool>false</bool>
     </property>
     <property name="isSavedStationsVisible" stdset="0">
      <bool>false</bool>
     </property>
     <property name="isHorisontal" stdset="0">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="4">
    <widget class="QDoubleSpinBox" name="sbSearchRange">
     <property name="maximum">
      <double>999999.000000000000000</double>
     </property>
     <property name="value">
      <double>15.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="2" column="5">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>км</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="9">
    <widget class="QLabel" name="lbStationList">
     <property name="text">
      <string>ВИДЖЕТ НЕ ИНИЦИАЛИЗИРОВАН!</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="11">
    <widget class="QTreeWidget" name="stationsList">
     <property name="styleSheet">
      <string notr="true">alternate-background-color: rgb(251, 244, 225);</string>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
     <property name="rootIsDecorated">
      <bool>true</bool>
     </property>
     <property name="uniformRowHeights">
      <bool>true</bool>
     </property>
     <property name="itemsExpandable">
      <bool>false</bool>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <property name="expandsOnDoubleClick">
      <bool>false</bool>
     </property>
     <attribute name="headerCascadingSectionResizes">
      <bool>true</bool>
     </attribute>
     <column>
      <property name="text">
       <string>Пункт?</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string/>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Название</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Широта, °</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Долгота, °</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Высота, м</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Тип станции</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Расстояние до пункта, км</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="2" column="8">
    <widget class="QPushButton" name="pbRemove">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>90</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>Удалить</string>
     </property>
    </widget>
   </item>
   <item row="2" column="6">
    <widget class="QPushButton" name="btnSearch">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>90</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>Найти</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QLabel" name="lbSearchRange">
     <property name="text">
      <string>Радиус поиска:</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>StationWidget</class>
   <extends>QWidget</extends>
   <header location="global">meteo/commons/ui/custom/stationwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
